src/main/kotlin/no/fdk/concept_catalog/service/SkosApNoModelService.kt

```diff
@@ -173,8 +176,12 @@ class SkosApNoModelService(
 
     private fun addScopeToDefinition(definitionBuilder: DefinitionBuilder, concept: Begrep) {
         concept.omfang
-            ?.takeIf { it.tekst?.isNotBlank() == true || it.uri?.isNotBlank() == true }
-            ?.let { definitionBuilder.scopeBuilder().label(it.tekst, NB).seeAlso(it.uri).build() }
+            ?.takeIf { it.tekst?.isNotBlank() == true || it.uri?.isValidURI() == true }
```

Comment:
Hva er forskjellen på det og dette?
```suggestion
            ?.takeIf { it.tekst?.isNotBlank() || it.uri?.isValidURI() }
```

Reply:
`it.tekst?` kan returnere `null`, mens `if`-testen krever verdier utelukkende av typen `true` eller `false`. Må derfor eksplisitt sjekke om returverdien fra `it.tekst?.isNotBlank()` er `true`.

Reply:
isValidURI returnerer alltid boolean så den trenger det ikke og du kan fint bytte ut isNotBlank med isNullOrBlank. ie:
```suggestion
            ?.takeIf { !it.tekst.isNullOrBlank() == true || it.uri?.isValidURI() }
```

Reply:
Sant! 👍